pkgs/development/python-modules/prometheus-client/default.nix

```diff
@@ -16,7 +16,7 @@ buildPythonPackage rec {
     owner = "prometheus";
     repo = "client_python";
     rev = "refs/tags/v${version}";
-    hash = "sha256-LabvQMNiRH4jclMnN0P4M3w25NQasNQEu1HWvRDQQ2o=";
+    sha256 = "sha256-FYQE0toy5VFKNVadSsxG/5NCRANYJOcVR4bGPrCAxvc=";
```

Comment:
Thanks, will update that.  

I do see a oblique reference to the preference for `hash` over `sha256` at https://nixos.wiki/wiki/Nix_Hash
I had used `sha256` since when I tried to compute the value using `lib.fakeSha256` it wasn't able to determine the has value.

Is there a better way of identifying the correct hash when using the `hash` attribute?